bindings/bind_py_feature_manager.cc

```diff
@@ -29,41 +29,82 @@
 
 using namespace rascal;  // NOLINT
 
+template <typename T>
+void bind_feature_manager_base(py::module & m_garbage) {
+  using Base_t = FeatureManagerBase<T>;
+
+  std::string featurebase_name = internal::GetBindingTypeName<Base_t>();
+  py::class_<Base_t>(m_garbage, featurebase_name.c_str());
```

Comment:
I still don't like the name `m_garbage` (could be confused with https://docs.python.org/3/library/gc.html) - try `m_throwaway` or `m_placeholder`?